src/transformStateWithClones.js

```diff
@@ -3,11 +3,47 @@
 /**
  * @param {Object} state
  * @param {Object[]} actions
- *
- * @return {Object[]}
+ * @returns {Object[]}
  */
-function transformStateWithClones(state, actions) {
-  // write code here
+function transformState(state, actions) {
+  const result = [];
+  let cloneOfState = { ...state };
```

Comment:
please read about let and const for objects, i think it could be const 😉

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#const_in_objects_and_arrays